addons/web/static/tests/core/hotkeys/hotkey_service_tests.js

```diff
@@ -527,3 +487,35 @@ QUnit.test("registrations and elements belong to the correct UI owner", async (a
     comp1.destroy();
     comp2.destroy();
 });
+
+QUnit.test("replace the overlayModifier", async (assert) => {
+    assert.expect(3);
+
+    const hotkeyService = serviceRegistry.get("hotkey");
+    patchWithCleanup(hotkeyService, {
+        overlayModifier: "alt+shift",
+    });
+
+    class MyComponent extends Component {
+        onClick() {
+            assert.step("click");
+        }
+    }
+    MyComponent.template = xml`
+        <div>
+        <button t-on-click="onClick" data-hotkey="b"/>
+        </div>
+    `;
+    await mount(MyComponent, { env, target });
+
+    const key = "b";
+    triggerHotkey(`alt+shift+${key}`);
+
+    await nextTick();
+    assert.verifySteps(["click"]);
+
+    triggerHotkey(`alt+${key}`);
+    await nextTick();
+
+    assert.verifySteps([]);
+});
```

Comment:
What about adding a test that asserts that overlay modifier can also be replaced and it still works on MacOS ?